app/services/distribution_service.rb

```diff
@@ -8,6 +8,9 @@ def perform_distribution_service(&block)
   rescue ActiveRecord::RecordNotFound => e
     Rails.logger.error "[!] #{self.class.name} failed to destroy distribution #{distribution_id} because it does not exist"
     set_error(e)
+  rescue ActiveRecord::RecordInvalid => e
+    Rails.logger.error  "[!] #{self.class.name} #{e} "
```

Comment:
```suggestion
    Rails.logger.error "[!] #{self.class.name} #{e} "
```

Looks like a bit of lint that Rubocop got mad at